tests/documentation/test_files/DocumentationStyleBear/bad_file.py.test.correct

```diff
@@ -0,0 +1,12 @@
+def hello_there(x):
+    """
```

Comment:
This is technically the first line.  So where the bear docs says "The first line needs to have no indentation." , it is not happening here.

This is one of the biggest doc style disagreements in python world : should the first text of the multiline docstring appear on the first line, immediately after the `"""`  or on the second line.

`pydocstyle` of course supports both modes.